lib/iris/experimental/um.py

```diff
@@ -405,11 +409,16 @@ def read_data(self):
                 data = np.fromfile(self.source, dtype, count=rows * cols)
                 data = data.reshape(rows, cols)
             elif lbpack == 1:
-                from iris.fileformats.pp_packing import wgdos_unpack
-                data_size = ((field.lbnrec * 2) - 1) * _WGDOS_SIZE
-                data_bytes = self.source.read(data_size)
-                data = wgdos_unpack(data_bytes, field.lbrow, field.lbnpt,
-                                    field.bmdi)
+                if mo_pack is not None:
+                    data_size = ((field.lbnrec * 2) - 1) * _WGDOS_SIZE
+                    data_bytes = self.source.read(data_size)
+                    data = mo_pack.unpack_wgdos(data_bytes, field.lbrow,
+                                                field.lbnpt, field.bmdi)
+                else:
+                    msg = 'iris.fileformats.pp_packing.wgdos_unpack has been ' \
```

Comment:
I don't think this is representative of the exception. ``mo_pack is needed to read packed data`` would be a suitable message.